### PR TITLE
change time/delay for qemu-guest-agent regarding my issue : output ip fail with bridge and I need it for my ansible inventory #731

### PR DESCRIPTION
**Pull request by itwars**
Move from Second to Minute

Please make sure you read [the contributor documentation](https://github.com/dmacvicar/terraform-provider-libvirt/blob/master/CONTRIBUTING.md) before opening a Pull Request.

### CODE REVIEW

File: libvirt/qemu_agent.go [hunk 1]
```diff
@@ -113,9 +113,9 @@ func qemuAgentGetInterfacesInfo(domain Domain, wait4ipv4 bool) []libvirt.DomainI
 		Pending:    []string{qemuGetIfaceWait},
 		Target:     []string{qemuGetIfaceDone},
 		Refresh:    qemuAgentInterfacesRefreshFunc(domain, wait4ipv4),
-		MinTimeout: 4 * time.Second,
-		Delay:      4 * time.Second, // Wait this time before starting checks
-		Timeout:    30 * time.Second,
+		MinTimeout:  1 * time.Minute,
+		Delay:      30 * time.Second, // Wait this time before starting checks
+		Timeout:    30 * time.Minute,
```
Comment (MalloZup) [inline libvirt/qemu_agent.go:118]: 30 Min is to much

Comment (MalloZup) [reply libvirt/qemu_agent.go:118]: we need 5 minutes

Comment (itwars) [reply libvirt/qemu_agent.go]: OK I have reduce to 5 min

**Comment by MalloZup** [comment]
@itwars did you checked if it fix your issue already? TIA

**Comment by itwars** [comment]
Yes it fixe my issue

**Comment by schlad** [comment]
Redhead ki7 > __

**Comment by itwars** [comment]
Hi, my patch worked with previous release. I'm back to business after some holidays, and again it's failing to get ip adresses!

**Pull request merged**